### PR TITLE
blog: relase 20.06

### PR DESCRIPTION
**Pull request by sduthil**

### CODE REVIEW

File: content/blog/sprint-review-2006.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+For more details about the aforementioned topics, please see the roadmap linked below.
+
+See you at the next sprint review!
+
+## Resources
+
+* [Install Wazo Platform](https://wazo-platform.org/install)
+
+* [Upgrade Wazo and Wazo Platform](/uc-doc/upgrade/). Be sure to read the [breaking changes](/uc-doc/upgrade/upgrade_notes#20.06)
```
Comment (chartrandf) [inline content/blog/sprint-review-2006.md:37]: This link anchor is not working.

```suggestion
* [Upgrade Wazo and Wazo Platform](/uc-doc/upgrade/). Be sure to read the [breaking changes](/uc-doc/upgrade/upgrade_notes#20-06)
```

File: content/blog/sprint-review-2006.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* **Phones**: We are working on synchronizing the DND status of phones with the status of Wazo, to avoid having two kinds of DND, one on the phone and another in Wazo applications.
+
+
+For more details about the aforementioned topics, please see the roadmap linked below.
+
+See you at the next sprint review!
+
+## Resources
+
+* [Install Wazo Platform](https://wazo-platform.org/install)
```
Comment (chartrandf) [inline content/blog/sprint-review-2006.md:35]:
```suggestion
* [Install Wazo Platform](/install)
```

File: content/blog/sprint-review-2006.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+* [Upgrade Wazo and Wazo Platform](/uc-doc/upgrade/). Be sure to read the [breaking changes](/uc-doc/upgrade/upgrade_notes#20.06)
+
+Sources:
+
+* [Upgrade notes](/uc-doc/upgrade/upgrade_notes#20.06)
```
Comment (chartrandf) [inline content/blog/sprint-review-2006.md:41]:
```suggestion
* [Upgrade notes](/uc-doc/upgrade/upgrade_notes#20-06)
```

File: content/blog/sprint-review-2006.md (hunk 1 above)
Comment (sduthil) [reply content/blog/sprint-review-2006.md]: fixed, thank you

File: content/blog/sprint-review-2006.md (hunk 2 above)
Comment (sduthil) [reply content/blog/sprint-review-2006.md]: fixed, thank you

File: content/blog/sprint-review-2006.md (hunk 3 above)
Comment (sduthil) [reply content/blog/sprint-review-2006.md]: fixed, thank you

**Pull request merged**